R/install.R

```diff
@@ -658,44 +658,16 @@ renv_install_postamble <- function(packages) {
 
   # only diagnose packages currently loaded
   packages <- renv_vector_intersect(packages, loadedNamespaces())
-  if (empty(packages))
-    return(TRUE)
 
-  # get version of package in library
```

Comment:
I realise it's slightly bad form to delete comments, but IMO based on the names, it's pretty obvious what the code is doing.